src/Inventory/Conf.php

```diff
@@ -935,38 +935,54 @@ public function showConfigForm()
             );
             //if action == action_status => show blocation else hide blocaction
             echo Html::scriptBlock("
-            function changestatus() {
-                if ($('#dropdown_stale_agents_action$rand').val() != 0) {
-                $('#blocaction1').show();
-                $('#blocaction2').show();
-                } else {
-                $('#blocaction1').hide();
-                $('#blocaction2').hide();
+                function changestatus() {
+                    if ($('#dropdown_stale_agents_action$rand').val() != 0 && $('#dropdown_stale_agents_action$rand').val() != 2) {
+                        $('#bloc_status_action1').show();
+                        $('#bloc_status_action2').show();
+                    } else {
+                        $('#bloc_status_action1').hide();
+                        $('#bloc_status_action2').hide();
+                    }
```

Comment:
Code can be simplified a bit

```suggestion
                    const display = $('#dropdown_stale_agents_action$rand').val() != 0 
                        && $('#dropdown_stale_agents_action$rand').val() != 2;
                       
                    $('#bloc_status_action1').toggle(display);
                    $('#bloc_status_action2').toggle(display);
```

Reply:
fixed

Reply:
Are you sure ? Code seems unchanged

Reply:
yes (sorry) I modified the code proposal, as the ```select``` allows ```mutliple``` selection the ```val()``` returns an ```array```